test/e2e/build_test.go

```diff
@@ -72,9 +72,9 @@ var _ = Describe("Podman build", func() {
 
 		st, err := os.Stat(logfile)
 		Expect(err).To(BeNil())
-		Expect(st.Size()).To(Not(Equal(0)))
+		Expect(st.Size()).To(Not(Equal(int64(0))))
```

Comment:
```suggestion
		Expect(st.Size()).To(Not(Equal(0)))
```

Reply:
@zhangguanzhang do you have any objections to merging this with the code as is?  I think the new test with int64 is valid.
LGTM